apps/sms/views/conversation/js/conversation.js

```diff
@@ -692,11 +692,14 @@ var ConversationView = {
     // Add recipients and fill+focus the Compose area.
     var recipientPromise;
     if (activity.contact && number) {
-      recipientPromise = Utils.getContactDisplayInfo(findByAddress, number)
-        .then(function onData(data) {
-          data.source = 'contacts';
-          return data;
-        });
+      recipientPromise = findByAddress(number).then((contacts) => {
+        return Object.assign(
+          Utils.basicContact(number, contacts),
+          { 
+            source: 'contacts' 
+          }
```

Comment:
you can write `{ source: 'contacts' }` on one line as this will be more readable in my opinion